Content/Infrastructure/Sulu/Sitemap/ContentSitemapProvider.php

```diff
@@ -81,19 +86,21 @@ public function __construct(
         string $kernelEnvironment,
         string $contentRichEntityClass,
         string $routeClass,
-        string $alias
+        string $alias,
+        int $pageSize = self::PAGE_SIZE
```

Comment:
I personally would no add it to the constructor:

```suggestion
        string $alias
```